test/passing/tests/attributes.ml

```diff
@@ -65,7 +65,7 @@ module type M = sig
 
   module T : [%ext] [@test7]
 
-  module T = T [@@test8]
+  module [@test8] T = T
```

Comment:
It would be nice to have both the new and old modules.
- To not suggest it's a regression
- Both are useful

Reply:
The test case is changed and I had to check it wasn't changed by ocamlformat. By "old" I meant the test case as it was before your changes.

Reply:
The new version of the test case is also useful, we should keep both.